cmd/settings.go

```
@@ -26,10 +26,14 @@ type Config struct {
 }
 
 func (c *Config) Load() error {
-	podmanPath, _ := exec.LookPath("podman")
-	if podmanPath != c.PodmanBinary && (podmanPath != "" || len(podmanPath) > 0) {
-		os.Setenv("PODMAN_BIN", podmanPath)
+	var envValue = os.Getenv("PODMAN_BIN")
+	if envValue == "" {
+		podmanPath, _ := exec.LookPath("podman")
```

Worried about eating the error here

It does seem that LookPath has some interesting edge case:

> In older versions of Go, LookPath could return a path relative to the current directory. As of Go 1.19, LookPath will instead return that path along with an error satisfying errors.Is(err, ErrDot). See the package documentation for more details.

Can you also add a check if err is not nil && err is not ErrDot? if it is error I think this means that Rodman is not installed and we can error out noting that. @pranavgaikwad and @eemcmullan will defer to y'all if I understood that correctly :)

---

I think this sounds smart. I don't remember the original intention behind masking the error here, but +1 on checking for those two.

---

I would like to concentrate on correct handling of the PODMAN_BIN variable here, as the bug #107 and the PR #108 is about that problem. If you want to check whether `podman` is installed at all and abort Kantra when this is not the case, this is a different concern. OK?

---

Considering that this was prior behavior, I think it is reasonable that we can do it in a follow up, sorry just caught it now, but that makes sense. 

Created a follow up issue: https://github.com/konveyor/kantra/issues/113